<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="5" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="31" started-at="2017-08-09T02:19:47Z" finished-at="2017-08-09T02:19:47Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="31" started-at="2017-08-09T02:19:47Z" finished-at="2017-08-09T02:19:47Z">
      <class name="co.edu.uelbosque.swii.calculadoraM.NewTest">
        <test-method status="PASS" signature="deboDividirCorrectamente()[pri:0, instance:co.edu.uelbosque.swii.calculadoraM.NewTest@396a51ab]" name="deboDividirCorrectamente" duration-ms="8" started-at="2017-08-08T21:19:47Z" finished-at="2017-08-08T21:19:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deboDividirCorrectamente -->
        <test-method status="PASS" signature="deboMultiplicarCorrectamente()[pri:0, instance:co.edu.uelbosque.swii.calculadoraM.NewTest@396a51ab]" name="deboMultiplicarCorrectamente" duration-ms="1" started-at="2017-08-08T21:19:47Z" finished-at="2017-08-08T21:19:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deboMultiplicarCorrectamente -->
        <test-method status="PASS" signature="deboRestarCorrectamente()[pri:0, instance:co.edu.uelbosque.swii.calculadoraM.NewTest@396a51ab]" name="deboRestarCorrectamente" duration-ms="1" started-at="2017-08-08T21:19:47Z" finished-at="2017-08-08T21:19:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deboRestarCorrectamente -->
        <test-method status="PASS" signature="deboSumarCorrectamente()[pri:0, instance:co.edu.uelbosque.swii.calculadoraM.NewTest@396a51ab]" name="deboSumarCorrectamente" duration-ms="1" started-at="2017-08-08T21:19:47Z" finished-at="2017-08-08T21:19:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deboSumarCorrectamente -->
        <test-method status="PASS" signature="seleccionarComandoCorrecto()[pri:0, instance:co.edu.uelbosque.swii.calculadoraM.NewTest@396a51ab]" name="seleccionarComandoCorrecto" duration-ms="1" started-at="2017-08-08T21:19:47Z" finished-at="2017-08-08T21:19:47Z">
          <exception class="co.edu.uelbosque.swii.calculadoraM.ComandoNoEncontradoException">
            <full-stacktrace>
              <![CDATA[co.edu.uelbosque.swii.calculadoraM.ComandoNoEncontradoException
	at co.edu.uelbosque.swii.calculadoraM.EncontrarComando.cmd(EncontrarComando.java:7)
	at co.edu.uelbosque.swii.calculadoraM.NewTest.seleccionarComandoCorrecto(NewTest.java:45)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- co.edu.uelbosque.swii.calculadoraM.ComandoNoEncontradoException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- seleccionarComandoCorrecto -->
      </class> <!-- co.edu.uelbosque.swii.calculadoraM.NewTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
